src/utils/envVars.ts

```diff
@@ -25,6 +25,12 @@ if(process.env.REACT_APP_ETH_REQUIREMENT && Number.isNaN(Number(process.env.REAC
 }
 export const ETH_REQUIREMENT            = process.env.REACT_APP_ETH_REQUIREMENT     || 524288;
 
+// ETH_DEPOSIT_OFFSET is added to the balance of the deposit contract to account for testnet deposit-contracts that allow some number of free deposit
+if(process.env.REACT_APP_ETH_DEPOSIT_OFFSET && Number.isNaN(Number(process.env.REACT_APP_ETH_DEPOSIT_OFFSET))) {
+    throw new Error("REACT_APP_ETH_DEPOSIT_OFFSET must be of type: number")
+}
+export const ETH_DEPOSIT_OFFSET = Number(process.env.REACT_APP_ETH_DEPOSIT_OFFSET) * Number(IS_MAINNET) || 0;
```

Comment:
Why multiply it with `IS_MAINNET`? Do you mean:
```suggestion
export const ETH_DEPOSIT_OFFSET = Number(process.env.REACT_APP_ETH_DEPOSIT_OFFSET) * Number(!IS_MAINNET) || 0;
```
?

(add a `!`)